### ISSUE TITLE
Zed crashes upon spawning task from keybinding

### ISSUE THREAD

**Issue by baldwindavid** [zed-industries/zed#10380]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

I noticed this immediately upon attempting to run a test under cursor with the latest preview. It appears that spawning any task via keybinding results in an immediate crash. Most minimal example task...

```json
{ "label": "Echo", "command": "echo hello" },
```

Keybinding in normal mode...

```json
"space t": [
  "task::Spawn",
  {
    "task_name": "Echo"
  }
],
```

The task works when directly spawning via command palette, just not via keymap.

### Environment

Zed: v0.131.0 (Zed Preview)
OS: macOS 14.4.1
Memory: 32 GiB
Architecture: aarch64

**Comment by SomeoneToIgnore** [comment]
Thank you for reporting, a fix (with the test to hopefully prevent it breaking again) is available in Zed Preview 0.131.1

**Comment by baldwindavid** [comment]
Awesome, works great. Thanks, @SomeoneToIgnore !
